ui/src/app/cohort-search/cohort-search.module.ts

```diff
@@ -0,0 +1,51 @@
+import {NgModule} from '@angular/core';
+import {CommonModule} from '@angular/common';
+import {ClarityModule} from 'clarity-angular';
+
+/* Components */
+// tslint:disable:max-line-length
```

Comment:
It may make sense to move the lint max-line-length lines to encapsulate all of the imports?